hack/verify-shellcheck.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Let's give credit to wherever we copy this from.
Similar to:
https://github.com/kubernetes-sigs/cluster-api/blob/d38a4b1c13b5f26ac407eab85b83597662f630bd/util/patch/patch.go#L214

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/3ead4407ea7d48535a429d433d15093566f53df6/hack/verify-shellcheck.sh#L19